docs/model-dev-guide/api-guides/apis-howto/api-core-ug.rst

```diff
@@ -508,13 +508,57 @@ In the Determined WebUI, go to the **Cluster** pane.
 You should be able to see multiple slots active corresponding to the value you set for
 ``slots_per_trial`` you set in ``distributed.yaml``, as well as logs appearing from multiple ranks.
 
+.. _core-profiling:
+
 ***********
  Profiling
 ***********
 
-Profiling with native profilers such as can be configured as usual. If running on a Determined
-cluster, the profiling log output path can be configured for automatic upload to the Determined
-Tensorboard UI.
+There are two ways to profile the performance of your training job:
+
+#. Core API's built-in system metrics profiler
+
+#. Integration with profilers native to your training framework, such as the Tensorflow and PyTorch
+profilers
+
+.. _core-profiler:
+
+Core API Profiler
+=================
+
+Core API comes with profiling functionality that tracks system metrics throughout the training run.
+These metrics are recorded at specified intervals to the master and can be viewed in the Web UI
+under your experiment's "Profiling" tab.
```

Comment:
The Core API includes a profiling feature that monitors and records system metrics during the training run. These metrics are recorded at specified intervals and sent to the master, allowing you to view them in the "Profiling" tab of your experiment in the WebUI.